### PR TITLE
Fix python lint issues

### PR DESCRIPTION
**Pull request by komalali**
# Description

Fixes lint issues highlighted by pylint 2.11.0

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works

- [ ] I have updated the [CHANGELOG-PENDING](https://github.com/pulumi/pulumi/blob/master/CHANGELOG_PENDING.md) file with my change

- [ ] Yes, there are changes in this PR that warrants bumping the Pulumi Service API version

### CODE REVIEW

**Comment by komalali** [comment]
@pgavlin I just fixed the issues that came up in https://github.com/pulumi/pulumi/pull/7986 - I'll just rebase and undo your pin once you merge your PR.

**Review by pgavlin — APPROVED** [review]
LGTM!

**Review by justinvp — APPROVED** [review]

**Comment by pgavlin** [comment]
@komalali gonna go ahead and merge this to unblock other PRs

**Pull request merged**
